testfiles/gedcom70/filename-1.ged

```diff
@@ -0,0 +1,21 @@
+﻿0 HEAD
+1 GEDC
+2 VERS 7.0
+0 @1@ OBJE
+1 FILE file:///unix/absolute
+2 FORM image/bmp
+1 FILE file:///c:/windows/absolute
+2 FORM image/bmp
+1 FILE file://windows/server
+2 FORM image/bmp
+1 FILE a/relative/path
+2 FORM image/bmp
+1 FILE a/relative/path
+2 FORM image/bmp
+1 FILE most/paths%3Fget%23escaped%5Blike%5Dthis
+2 FORM image/bmp
+1 FILE https://leave.alone?with=args#and-frags
+2 FORM image/bmp
+1 FILE https%3a/not.a.url/even-though-similar
+2 FORM image/bmp
```

Comment:
These three examples were testing my sample converter's handling of % escapes, which are not addressed in the spec except insofar as they are in the referenced specs [RFC 3986](https://www.rfc-editor.org/info/rfc3986) and the [WHATWG URL specification](https://url.spec.whatwg.org/). I feel like some kind of clarifying note might make this example clearer, but there's no FILE.NOTE in the spec so that would require some refactoring of this file.